README.md

```diff
@@ -28,6 +28,12 @@ var obj = {
 fixturify.writeSync('testdir', obj) // write it to disk
 
 fixturify.readSync('testdir') // => deep-equals obj
+
+fs.removeSync('testDir', {
+    'subdir': { 'bar.txt' }
+  }) // remove subdir/bar.txt
```

Comment:
> I assume you meant fixturify.removeSync?

fixed the type `fs.removeSync` -> `fixturify.removeSync`

> And { 'bar.txt' } doesn't look right...

Yup, i should have just copy/past'd from my tests :P